pkg/trace/pb/hook_test.go

```diff
@@ -22,13 +22,28 @@ func TestMetaHook(t *testing.T) {
 		s, err := decodeBytes(b)
 
 		assert := assert.New(t)
+		assert.False(HasMetaHooks())
 		assert.Nil(err)
 		assert.Equal(map[string]string{"card.number": "4166 6766 6766 6746"}, s.Meta)
 	})
 
-	t.Run("on", func(t *testing.T) {
-		SetMetaHook(func(k, v string) string { return "test" })
-		defer SetMetaHook(nil)
+	t.Run("meta struct off", func(t *testing.T) {
```

Comment:
You can merge this test into the previous one and just have one test called `off` which tests behaviour without any hooks enabled.